src/features/structure/Header.tsx

```diff
@@ -3,8 +3,24 @@ import AirSwapLogoWithText from "../../assets/airswap-logo-with-text.svg";
 import { SettingsMenuButton } from "../settings/SettingsMenuButton";
 import { WalletConnection } from "../chain-connection/WalletConnection";
 import { StakeButton } from "../staking/StakeButton";
+import { useEffect, useState } from "react";
+import WalletConnectModal from "../../components/WalletConnectModal";
+import { useAccount } from "wagmi";
+
+export const Header = ({ }: {}) => {
+  const [isRenderWalletConnectModal, setIsRenderWalletConnectModal] = useState<boolean>(false)
```

Comment:
extremely minor:

`isRender` is a bit weird.. I'd just use `show`, i.e. `showWalletConnectModal` and `setShowWalletConnectModal`